src/imports/facepile.njk

```diff
@@ -2,7 +2,7 @@
 permalink: /imports/facepile.html
 ---
 <div class="facepile">
-{%- for author in authors | values | shuffle(110) %}
+{%- for author in authors | values | shuffle(63) %}
```

Comment:
Just out of curiousity. Where does this magic number come from?

Reply:
I think it’s simply 3 rows at the max-width